### DISCUSSION TITLE
NIM_Duilib未来发展问题

### DISCUSSION THREAD

**Discussion by 204065248** [netease-im/NIM_Duilib_Framework#438] [General]
大厂的人员变动很频繁，这个大家都懂。之前听到有2.0计划。但是距离上个版本已经相隔3年了，我个人还是很喜欢网易版的duilib，所以我想知道，2.0版本是否存在，有没有打算做的跟饿了么那种一样的文档，还有UI设计器【这方面有个个体开发作者的炫彩设计器算蛮不错的】。我相信大部分C++开发者都是苦于UI上面，不像C#的UI库百花齐放。麻烦解答一下~ @nmgwddj

**Comment by nmgwddj** [comment]
首先，感谢您关注 NIM duilib 的情况。
在我们计划内，duilib 可能首先支持跨平台的渲染引擎如 Skia。但从现在的工作中心来看，我不得不以个人的角度承认 UI 框架上的资源正在往其他方面倾斜。但这并不代表我们不会再在 duilib 上发力。
NIM duilib 的定位不期望大而全，是小并且敏捷。任何一个开源框架的发展都离不开真实的使用者和社区成员（可能是双重角色）的支持和贡献。从这一点来看可能部分维护者已经“不称职”。我们期望有这方面开发意向的开发者与我们一起审议新的提案并逐步实现它，这可能不是功利性的。我们也期望能创造这样一种氛围从一个小的 UI 框架做起。

**Reply by 204065248** [reply]
> 

是的 这个我明白，我觉得NIM duilib很好用，但是如果以后能够有配套的 UI设计器就更好了。就比如我上面所说的炫彩UI设计器，极大程度的提高了UI设计效率

**Reply by fyozt** [reply]
我也比较期待网易后期对 NIM duilib 的更新支持
